__tests__/errors/AppiError.test.js

```diff
@@ -0,0 +1,53 @@
+const {
+  ApiError,
+  BadRequest,
+  Unauthorized,
+  Forbidden,
+  NotFound,
+  Conflict,
+  ServerError
+} = require("../../errors/ApiError");
+
+describe("eroor's functions", () => {
+  it("should return new instance for ApiError class with 2 public properies", () => {
+    const obj = new ApiError("Something went wrong", 418);
+    expect(obj.message).toBe("Something went wrong");
+    expect(obj.status).toBe(418);
+  });
+
+  it("should return new instance for BadRequest class with 2 public properies", () => {
+    const obj = new BadRequest("Something went wrong");
+    expect(obj.message).toBe("Something went wrong");
+    expect(obj.status).toBe(400);
+  });
```

Comment:
it("should return new instance for BadRequest class with 2 public properies", () => {
    
    const errorMessage = "Something went wrong"

    const obj = new BadRequest(errorMessage);
    expect(obj.message).toBe(errorMessage);
    expect(obj.status).toBe(400);
  });

Reply:
already fixed